src/main/fc/fc_msp.c

```diff
@@ -142,13 +142,13 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXSONAR, "SONAR", 22 },
     { BOXSERVO1, "SERVO1", 23 },
     { BOXSERVO2, "SERVO2", 24 },
-    { BOXSERVO3, "SERVO3", 25 },
     { BOXBLACKBOX, "BLACKBOX", 26 },
     { BOXFAILSAFE, "FAILSAFE", 27 },
     { BOXAIRMODE, "AIR MODE", 28 },
-    { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
-    { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
+    { BOX3DPOSSWITCH, "POSTIVE ONLY 3D", 29},
+    { BOX3DNEGSWITCH, "NEGITIVE ONLY 3D", 30},
```

Comment:
> I'm not a fan as much as using the mixer to do this. BOX3DDISABLESWITCH could have been done entirely in the mixer as well. Maybe a separate feature could be added: 

What are your arguments against using the mixer for this?

This would cause RC resolution to be halved when 3D is disabled, which is probably not what the author was aiming for. But I doubt that not having full resolution just for the case of flipping over a crashed craft will be a problem.